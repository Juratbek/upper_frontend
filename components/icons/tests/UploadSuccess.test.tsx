import { render, screen } from '@testing-library/react';

import { UploadSuccessIcon } from '../UploadSuccess';

describe('UploadSuccess icon', () => {
  it('renders the icon', () => {
    render(<UploadSuccessIcon />);
    const icon = screen.getByRole('icon');
    expect(icon).toMatchSnapshot();
  });
});
