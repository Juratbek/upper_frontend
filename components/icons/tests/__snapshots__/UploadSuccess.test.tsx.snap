// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UploadSuccess icon > renders the icon 1`] = `
<svg
  fill="none"
  height="20"
  role="icon"
  viewBox="0 0 24 24"
  width="20"
>
  <g>
    <path
      d="M15 11L11 15L9 13M23 15C23 12.7909 21.2091 11 19 11C18.9764 11 18.9532 11.0002 18.9297 11.0006C18.4447 7.60802 15.5267 5 12 5C9.20335 5 6.79019 6.64004 5.66895 9.01082C3.06206 9.18144 1 11.3498 1 13.9999C1 16.7613 3.23858 19.0001 6 19.0001L19 19C21.2091 19 23 17.2091 23 15Z"
      stroke="#4BE732"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="1"
    />
  </g>
</svg>
`;
